app/helpers/application_helper.rb

```diff
@@ -1,11 +1,5 @@
 module ApplicationHelper
-
-  # def rate_video(selected=nil)
-  #   options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]}, selected)
-  # end
-
-
-  def rate_video(selected=nil)
+ def rate_video(selected=nil)
```

Comment:
I would call this `video_rating_options`, since calling the method does not actually rate the video.